packages/salesforcedx-vscode-apex/snippets/apex.json

```diff
@@ -0,0 +1,176 @@
+{
+    "Class": {
+        "prefix": "class",
+        "body": [
+            "class ${1:Name}",
```

Comment:
Do we need to have a modifier? I'm not sure if the code will compile without a default modifier like public|global